### PR TITLE
Fix sending grid jobs to remote HTCondor-CEs

### PR DESCRIPTION
**Pull request by matyasselmeci**
Condor needs to bind to a public interface or else it can't send jobs to
HTCondor-CEs that are not on the same machine as the condor-cron
instance. Fix that, and use security controls to lock it down as much as
possible. This is for SOFTWARE-2267.

### CODE REVIEW

File: etc/condor_config [hunk 1]
```diff
@@ -54,20 +54,24 @@ MAIL         = /bin/mail
...
-#SEC_NEGOTIATOR_INTEGRITY = REQUIRED
-#SEC_NEGOTIATOR_AUTHENTICATION_METHODS = PASSWORD
-#SEC_CLIENT_AUTHENTICATION_METHODS = FS, PASSWORD
+SEC_DAEMON_AUTHENTICATION = REQUIRED
+SEC_DAEMON_INTEGRITY = REQUIRED
+SEC_DAEMON_AUTHENTICATION_METHODS = FS
+SEC_CLIENT_AUTHENTICATION_METHODS = FS, GSI
+SEC_DEFAULT_AUTHENTICATION = REQUIRED
+ALLOW_ADMINISTRATOR = $(CONDOR_ADMIN)
+ALLOW_OWNER = $(FULL_HOSTNAME)
```
Comment (bbockelm) [inline etc/condor_config:72]: This would allow any user logged into the host to shut down the service, right?

Comment (matyasselmeci) [inline etc/condor_config:72]: How? The obvious method doesn't work:

```
$ CONDOR_CONFIG=/etc/condor-cron/condor_config condor_off
ERROR
SECMAN:2010:Received "DENIED" from server for user matyas@fermicloud152.fnal.gov using method FS.
Can't send Kill-All-Daemons command to local master
```

**Comment by brianhlin** [comment]
@bbockelm suggested using `C_GAHP_WORKER_THREAD.BIND_ALL_INTERFACES` or `C_GAHP_WORKER_THREAD.NETWORK_INTERFACE` but neither of them seem to work. I don't believe that either of those knobs can be set on a per subsystem basis. I'll try to talk to Jaime to see if there are any other possible solutions.

**Comment by brianhlin** [comment]
We should be able to close this in favor of #2.

**Pull request closed without merging**